key.ui/src/main/java/de/uka/ilkd/key/gui/MainWindow.java

```diff
@@ -915,40 +915,50 @@ private JMenu createSelectionMenu() {
         return goalSelection;
     }
 
-    private JMenu createProofMenu() {
+    /**
+     * Create the proof menu.
+     *
+     * @param proof a specific proof that the menu should work on, may be null
```

Comment:
JavaDoc parameter name is already out of sync...

Reply:
Also done